README.md

```diff
@@ -10,25 +8,30 @@ The Hats protocol is designed to give white hats hackers the opportunity to gain
 
 ### Usage
 
+Installation:
 
- Installation:
+```
+npm install
+```
 
- ```
- npm install
- ```
+Create `.env` and `.private.json` files as needed.
```

Comment:
This is duplicate and very confusing, better have just one (probably .env) and remove the other file

Reply:
yes